Update Readme (#161)
immuni-app/immuni-app-android: README.md

# Checking the build

In addition, to make the code open-source, we support people to verify that every build published on the App Store comes from a new commit stored in this repository. Please refer to the [Immuni Technology Description](https://github.com/immuni-app/documentation/blob/master/Technology%20Description.md#android-app-technologies) for a complete overview of the goals and status of this effort.

> Currently we provide an open and continuous integration for building and improving the client. [Here](.circleci/config.yml) is the full specification. Every new release will include, reported as a new Issue, what we have done and what we suppose to improve and implement.

Reviewer: Can someone confirm the last part of this translation is/isn't ok? It doesn't look right to me

# Contributing

> Contributions are really appreciated and more than welcome. Before proceeding, please read carefully the [Code of Conduct](CODE_OF_CONDUCT.md) and consider it as an "How-To" guide to contribute the community and create a positive and collaborative environment. Additionally, please read also our [CONTRIBUTING](CONTRIBUTING.md) file that provides all info to make the contribution process easy.

Reviewer: ```suggestion
Contributions are welcome. Before proceeding, please read carefully the [Code of Conduct](CODE_OF_CONDUCT.md) and consider it as an "How-To" guide to contribute and have a positive approach to the community in order to create a collaborative environment. Additionally, please read also our [CONTRIBUTING](CONTRIBUTING.md) file that provides all info to make the contribution process easy.
```

> The Immuni project consists of several different repositories (one per component or service). Please consider this repository only related to Immuni for iOS. To propose or request a new feature, please open an issue here [Documentation repository](https://github.com/immuni-app/immuni-documentation). 

Reviewer: ```suggestion
The Immuni project consists of several different repositories (one per component or service). Please consider this repository only related to Immuni for Android. To propose a new feature, please open an issue here [Documentation repository](https://github.com/immuni-app/immuni-documentation).
```

> This will make everyone feel involved to see it, consider it, and participate to the discussion. Any opened issue or pulled request could slow down the overall process.

Reviewer: ```suggestion
This will make everyone feel involved to see it, consider it, and participate to the discussion. Any open issue or pull request could slow down the overall process.
```
The last phrase is confusing and is actually discussed #300
